var/spack/repos/builtin/packages/bedops/package.py

```diff
@@ -15,7 +15,8 @@ class Bedops(MakefilePackage):
     homepage = "https://bedops.readthedocs.io"
     url      = "https://github.com/bedops/bedops/archive/v2.4.39.tar.gz"
 
-    version('2.4.40', sha256='8c01db76669dc58c595e2e1b9bdb6d462f3363fc569b15c460a63a63b8b6bf30')
+    maintainers = ['jacorvar']
+    version('2.4.40', sha256='0670f9ce2da4b68ab13f82c023c84509c7fce5aeb5df980c385fac76eabed4fb')
```

Comment:
This is an invalid sha256 BUT #26141 has already been merged and it contains the maintainer and correct sha256 for this version.